adapter/__init__.py

```diff
@@ -5,5 +5,5 @@
 
 # build dist backup: drive, Public Code/Adapter
 
-__version__ = "1.7.0"
+__version__ = "1.7.1"
```

Comment:
@rhosbach - thanks for thinking of the version update. As I saw this I realized that this is a new feature rather than a patch, so we should increase the minor version, therefore use `1.8.0`.

Reply:
Gotcha.  I just pushed up that change.